pubpub/platform
# dev: add way to run easy end to end tests (temporarily)

## Issue(s) Resolved

Resolves #490 somewhat.

It is not as robust as what #490 describes, maybe this is more something @kalilsn can use as a starting guide to fulfill that ticket.

## Test Plan

1. Create a `.env.production.local` in `core` by doing `cp core/.env.development .env.production.local`
2. Run `pnpm playwright:test`
3. You should see that the app starts building, and that the tests start running (and hopefully pass)
4. Run `pnpm playwright:test:dev`
5. The app should not build but start in dev mode, tests should run and pass (although some errors might show up in the console)

## Screenshots (if applicable)

## Optional

I configured this in `playwright.config.ts`, no external script necessary.

This will need to be rewritten somewhat when we do E2E on CI, but for now I think this is fine. 

### Improvements

Maybe we should run `prisma reset` before (and after) running the test, or run a specific seed script.

### Notes/Context/Gotchas

### Supporting Docs
